actuator-firmware/src/analog_input.c

```diff
@@ -0,0 +1,34 @@
+#include <ch.h>
+#include <hal.h>
+
+#include "hal_adc.h"
+#include "hal_adc_lld.h"
+
+/* We have an external attenuation of 0.5 before reaching the analog input */
+static const float ADC_GAIN = 2 * 3.3 / 4096;
+
+static const ADCConversionGroup group = {
```

Comment:
This looks correct. I didn't go through the datasheet though.

Reply:
ADC configs are very error-prone, so I expect some mistakes to show up in there anyways :smile: